src/Common/Common.Client/IPowerBIClientCmdletInitFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+
+namespace Microsoft.PowerBI.Common.Client
+{
+    public interface IPowerBIClientCmdletInitFactory : IPowerBICmdletInitFactory
+    {
+        IPowerBIClientFactory Client { get; }
```

Comment:
I'm extending IPowerBIClientCmdletInitFactory to have a Client property. I don't want to make Client a part of IPowerBIClientCmdletInitFactory as the profile cmdlets don't need it and it would push up the dependency of PowerBI.API